docs/Installation.md

```diff
@@ -291,6 +291,19 @@ Initialize the database:
 mysql --user=root --password < ./initial-mysql.sql
 ```
 
+For latest versions of MySQL, wherein the root password is blank and you get an
+error "Access denied for the above command", follow the below procedure:

```

Comment:
Do you get this from a clean install? If so, on which OS versions? I don't believe I've seen this behavior.

Reply:
Yes Debian 9

Reply:
@mattias-p I have tested thrice now. I am getting the same results. 

Did you test?